app/models/miq_web_service_worker.rb

```diff
@@ -17,7 +17,10 @@ def self.supports_container?
   end
 
   def self.bundler_groups
-    %w[manageiq_default graphql_api]
+    # TODO: The api process now looks at the existing UI session as of: https://github.com/ManageIQ/manageiq-api/pull/543
+    # ui-classic should not be but is serialializing its classes into session, so we need to have access to them for deserialization
+    # sandboxes;FC:-ActiveSupport::HashWithIndifferentAccess{I"dashboard;FC;q{I"perf_options;FS:0ApplicationController::Performance::Options$typ0:daily_date0:hourly_date0: days0:
+    %w[manageiq_default ui_dependencies graphql_api]
```

Comment:
@skateman See above... because we serialize UI classic classes into session and the web service / api worker can read this existing UI session, it also need to know about UI classic classes ☹️ 

For now, I'm adding ui dependencies to the web service worker but we shouldn't have to pull in all of ui classic for this.